pkg/storage/v2/bigquery/writer/metrics.go

```diff
@@ -103,7 +103,7 @@ func getCodeFromError(err error) string {
 	return codeUnknown
 }
 
-func registerMetrics(r metrics.Registerer) {
+func RegisterMetrics(r metrics.Registerer) {
```

Comment:
Do we need to make this public?

Reply:
Yes, since the events dwh persisters need to use bigquery writer, but we can't register the metrics twice, so we have to register bigquery writer metrics when [the batch service starting](https://github.com/bucketeer-io/bucketeer/pull/952/files#diff-9f3bffbf9b659d0228f7f25aeb18a14d9fa79a72e2a8100e7371b1d112faa00cR685)